### PR TITLE
Cgprod 801 sign in

### PR DESCRIPTION
**Pull request by alex-s2**
Initial empty account object

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -64,9 +64,11 @@ define(['storage','websockets'], function(storage, ws) {
     var container = document.getElementById(gmi.gameContainerId);
     var wrapper = document.createElement("div");
     var inner = document.createElement("div");
+    var inner2 = document.createElement("div");
     wrapper.className = "wrapper";
     wrapper.tabIndex = -1;
     inner.className = "inner";
+    inner2.className = "inner2";
```
Comment (binhbui) [inline src/main.js]: why do you need an inner2?

Comment (alex-s2) [reply src/main.js]: oops that inner2 stuff was just testing, removed it locally but guess it didn't get committed, good spot.

File: src/index.html [hunk 2]
```diff
@@ -25,7 +25,7 @@
 <body>
     <div id="local-game-holder"></div>
 
-    <script src="http://static.bbc.co.uk/playpen/1.2.8/script/require/require.js"></script>
+    <script src="https://static.bbci.co.uk/frameworks/requirejs/lib.js"></script>
```
Comment (Leonie-) [inline src/index.html]: Do you need to make these changes to `index.dev.html` too?

Comment (Leonie-) [reply src/index.html]: Sorry, ignore my above comment!

**Pull request merged**